tool/jt.rb

```diff
@@ -772,17 +772,14 @@ def test_mri(*args)
   private :test_mri
 
   def test_compiler(*args)
-    jruby_opts = []
-
+    env = {}
+    
+    env['TRUFFLE_RUBY_OPTS'] = '-Xexceptions.print_java=true'
```

Comment:
Yeah it's quite long, but at least it's consistent with `RUBYOPT`.
Or should we only use `RUBYOPT`?